__tests__/Guides.unit.test.ts

```diff
@@ -1,10 +1,388 @@
+import Gesto from 'gesto';
+import { defaultProps, GUIDE } from '../src/react-guides/consts';
 import Guides from '../src/react-guides/Guides';
-import { GuidesProps } from '../src/react-guides/types';
+import { GuidesProps, GuidesState, LockGuides } from '../src/react-guides/types';
+import Ruler, { RulerProps } from '../src/react-ruler';
+
+const defaultGuides: number[] = [];
+const indexOfDeletedGuide = 2;
+let guidesTest: number[] = [5, -6, 3, -4.2];
+const deletedGuide = guidesTest[indexOfDeletedGuide];
+let selectedGuide: number = guidesTest[1];
+let eventMouse: MouseEvent = new MouseEvent('click');
+let isDragging = false;
 
```

Comment:
I don't understand why you set variables here and use a function to rewrite them globally.
The function should always be pure.
if you need to use this variables will be better to return them from function